docs/writing-tests/python-handlers/index.md

```diff
@@ -0,0 +1,25 @@
+# Python Handlers
+
+Python file handlers are designed to provide a vaguely PHP-like interface
+where each resource corresponds to a particular python file on the
+filesystem. Typically this is hooked up to a route like ``("*",
+"*.py", python_file_handler)``, meaning that any .py file will be
+treated as a handler file (note that this makes python files unsafe in
+much the same way that .php files are when using PHP).
+
+Unlike PHP, the python files don't work by outputting text to stdout
```

Comment:
Let's not quite so directly reference PHP (which isn't so useful if you don't know what PHP does!), but instead say something more like, "These Python files are assumed to define a single function `main(request, response)`. Note therefore anything written to stdout is not returned as part of the response, as is the case in PHP.